salt/config/__init__.py

```diff
@@ -3948,6 +3948,14 @@ def apply_master_config(overrides=None, defaults=None):
 
     opts = defaults.copy()
     opts["__role"] = "master"
+
+    # Suppress fileserver update in FSChan, for LocalClient instances generated
```

Comment:
@s0undt3ch @twangboy I know you've both already approved the PR, but this line should say FSClient instead of LocalClient. Mind if I change it before merging? Feel free to also just make this change yourself.

Reply:
Sure, go ahead a commit the change.